src/tests/ftest/util/server_utils_params.py

```diff
@@ -371,7 +371,8 @@ def __init__(self, index=None):
                 "ABT_MAX_NUM_XSTREAMS=100",
                 "DAOS_MD_CAP=1024",
                 "DD_MASK=mgmt,io,md,epc,rebuild",
-                "D_LOG_FILE_APPEND_PID=1"
+                "D_LOG_FILE_APPEND_PID=1",
+                "COVFILE=/tmp/test.cov"
```

Comment:
One thing to keep in mind here is this only will be included in tests that use the default `/run/server_config/servers[/x]/env_vars:` setting.  We have 21 test yaml files that override this setting in which case this new setting (and probably others - intentional or not) will not be set.  If we want to have environmental variables that are always set we need to define some different code.